src/render/index.js

```diff
@@ -65,7 +107,10 @@ const getFormat = (input) => {
  */
 const typeset = async (data) => {
     try {
-        return await MathJax.typeset(data);
+        const outputFormat = getOutputFormat(data.mml, data.svg);
+        const typesetFunction = CONVERSION_TABLE[`${data.format}_to_${outputFormat}`];
+        await MathJax.startup.promise;
+        return await typesetFunction(data.math);
```

Comment:
While I'm usually a fan of this type of map, in this particular case I think I'd find somehow more readable something like:

```suggestion
        await MathJax.startup.promise;
        switch (data.format) {
            case 'TeX':
            case 'inline-TeX':
                if (data.mml) {
                    return MathJax.tex2mmlPromise(data.math);
                }
                if (data.svg) {
                    return MathJax.tex2svgPromise(data.math);
                }
                throw new Error(`Supported output formats for ${data.format} input are: MathML, SVG`);

            case 'MathML':
                if (data.svg) {
                    return MathJax.mathml2svgPromise(data.math);
                }
                throw new Error(`Supported output formats for ${data.format} input are: SVG`);

            default:
                throw new Error(`Unsupported input format: ${data.format}`);
        }
```

(This isn't really a requested change… it's more like a matter of personal taste.)